src/multiplayer-servers/multiplayer-services/server-protection/gateway-policies.md

```diff
@@ -0,0 +1,55 @@
+# Gateway Policies
+
+Gateway policies provide an additional layer of protection for your game infrastructure by 
+redirecting specific outgoing game server traffic to destinations such as your game backend.
+
+If SteelShield is active, it might also block legitimate connection attempts or responses 
+due to its lack of awareness of your legitimate outgoing traffic.
+
+Implementing Gateway Policies offers an effective solution to prevent both issues.
+
+## Attack Scenario
+
+Without gateway policies, an attacker can spoof a game host's IP address and send malicious requests to your game backend. 
+In response, you or your hosting provider might decide to block the IP address. 
+As a result, your game host, including legitimate game server traffic, 
+would no longer be able to communicate with your game backend.
+
+By redirecting traffic from your game server through the gateway, 
+the traffic originates from gateway IP addresses that are typically unknown to attackers. 
+This makes spoofing a game host's IP address largely ineffective.
```

Comment:
largely? Does that mean it could still be effective in some situations? :p Either elaborate or do not mention it imo

Reply:
See "Exceptions". Some traffic might leave through the node. Still recommend to remove it?